debug_module.tex

```diff
@@ -299,6 +299,14 @@ \section{System Bus Access} \label{systembusaccess}
 involving a hart, regardless of whether Program Buffer is implemented.
 The System Bus Access block uses physical addresses.
 
+If a hart has a cache, the data in the cache may not be consistent with the
```

Comment:
I suggest leaving out all mention of "caches" and keeping it more abstract.

Depending on the microarchitecture, the data read and/or written by the debugger may not always be consistent with that observed by the hart(s). It is up to the debugger to enforce consistency if the implementation does not. This specification does not define a standard way to do this, as it is implementation/platform specific. Possibilities may include using the SystemBus and/or Program Buffer to write to special memory-mapped locations, or executing special instructions via the Program Buffer.